v20.2/string.md

```diff
@@ -33,6 +33,10 @@ When inserting a string:
 - If the value is cast as a string with a length limit (e.g., `CAST('hello world' AS STRING(5))`), CockroachDB truncates to the limit.
 - If the value is under the column's length limit, CockroachDB does **not** add padding. This applies to `STRING(n)` and all its aliases.
 
+{{site.data.alerts.callout_info}}
+<span class="version-tag">New in v20.1:</span> To improve compatibility with PostgreSQL, CockroachDB truncates trailing space characters in `STRING` variables.
```

Comment:
should only be CHAR column types, right?

Reply:
Hmm... I'm a little confused then. I thought CHAR was just an alias for STRING? That's how we document CHAR, at least: https://www.cockroachlabs.com/docs/stable/string.html#aliases

If they are two different data types, then we really need to update the docs.